client/verta/tests/test_versioning/test_dataset.py

```diff
@@ -60,6 +60,66 @@ def test_s3_no_duplicates(self):
 
         assert len(multiple_dataset._msg.s3.components) == len(bucket_dataset._msg.s3.components)
 
+    def test_versioned_bucket(self):
+        s3 = pytest.importorskip("boto3").client('s3')
+        S3_PATH = verta.dataset.S3._S3_PATH
+
+        bucket = "verta-versioned-bucket"
```

Comment:
I'm assuming you already put some data there.

Reply:
There is data there! I almost forgot:
@ravishetye fyi, you had created this bucket to test versioning with. I'm now using this bucket for our automated Client tests. If this would pose any problem for you, I can create a separate bucket.

Reply:
go for it. just a related datapoint when we version stuff and delete the object, the object is not actually deleted. We need to delete the version to keep the s3 cost in control.